### ISSUE TITLE
New command line processing anomaly - missing value

### ISSUE THREAD

**Issue by tbpassin** [leo-editor/leo-editor#3386]
With PR [3382](https://github.com/leo-editor/leo-editor/pull/3382) and the "--option" form:

```
C:\Users\tom>py-leo-git --theme
Using PYTHONPATH: ;c:\Tom\git\leo-editor
Invalid option: expected --theme=VALUE
```

but

`C:\Users\tom>py-leo-git --theme=`

runs with no error message.

I expect to see the same error message as the first example.

**Comment by edreamleo** [comment]
@tbpassin Thanks for your great testing.  I'll fix this immediately.

**Comment by edreamleo** [comment]
Closed via PR #3388.

**Comment by tbpassin** [comment]
Not quite there:

This works as intended:
```
C:\Users\tom>py-leo-git --theme .leo\workbook.leo
Invalid option: expected --theme=VALUE
```

but this still runs with no error message:
```
C:\Users\tom>py-leo-git --theme=
```

```
Leo 6.7.4-devel, devel branch, build a4db62efc6
2023-06-16 08:06:59 -0500
Python 3.11.4, PyQt version 6.4.3
Windows 10 AMD64 (build 10.0.19045) SP0
```

**Comment by edreamleo** [comment]
Fixed via [6e460e1](https://github.com/leo-editor/leo-editor/commit/6e460e1c7b6224d92a2dcb3fc25d7608257a3462) in devel.

Keep on testing!

**Comment by edreamleo** [comment]
@tbpassin Thanks for this comment. I have some thoughts for other tweaks. Watch for a new PR :-)
